### PR TITLE
Update atomix client and deps 

### PR DESCRIPTION
**Pull request by adibrastegarnia**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -13,23 +13,21 @@ require (
...
-	github.com/stretchr/testify v1.7.1
+	github.com/stretchr/testify v1.8.0
 	golang.org/x/net v0.0.0-20220412020605-290c469a71a5
 	golang.org/x/oauth2 v0.0.0-20220411215720-9780585627b5
 	google.golang.org/grpc v1.46.0
 	google.golang.org/protobuf v1.28.0
 	gopkg.in/yaml.v2 v2.4.0
 	gotest.tools v2.2.0+incompatible
-	k8s.io/client-go v0.22.1
+	k8s.io/client-go v0.24.0
```
Comment (adibrastegarnia) [inline go.mod:30]: this is problematic when I run integration tests.

**Comment by SeanCondon** [comment]
is this outdated now with your other patch?

**Comment by adibrastegarnia** [comment]
> is this outdated now with your other patch?

No I still need to work on it. We decided to move test and benchmark to separate modules to avoid having deps to k8s client. It was becoming nightmare to update atomix client in the code. I will do it in the next few days. It was falling behind a bit.

**Comment by adibrastegarnia** [comment]
perhaps I am gonna open a new PR to avoid conflicts

**Comment by adibrastegarnia** [comment]
retest this please

**Pull request merged**